lib/requests/foods.js

```diff
@@ -52,20 +62,20 @@ const getEachFood = (foods) => {
 }
 
 const renderFood = (food) => {
-  $('#food-info').prepend(`
-    <tr>
-      <td class="food-name">${food.name}</th>
-      <td class="food-calories">${food.calories}</th>
-      <td>
-      <button data-food-id="${food.id}" class="delete-btn" aria-label="Delete">
-        delete me
-      </button>
-      </td>
-    </tr>
-    `)
+  $('#food-table-info').prepend(
+   `<article class="food-item-${food.id}" id="food-item-row">
+      <p class="food-item-name">${food.name}</p>
+      <p class="food-item-calories">${food.calories}</p>
+      <div class="button-container">
+        <button id="${food.id}" class="food-item-delete-btn" aria-label="Delete">-</button>
```

Comment:
In class, we talked about the downsides of setting the id attribute of an HTML element to the single id. Each id of the element must be different, and Katelyn/Amy suggested adding a data attribute.

`
<button data-food-id="food-${food.id}".....>
`

This would then impact the way we pass information into the deleteFood function since it has a parameter of id passed in.

Reply:
I agree and see where such an overlap even might occur. I think it would be a good idea to change the code to incorporate a data attribute. I'll have to add to the function to pull out just the id.